azavea/echo-locator
# Update README with account info and yarn command

## Overview

Originally, `README.md` had the wrong yarn command in its section on 'Running directly' and did not provide any information on how to log in once spun up. This PR changes `yarn add` to `yarn install` and adds where to find log in credentials, as well as how to create a Client ID following log in.

## Testing Instructions

 * Try to run `yarn add` on host; should respond with error `Missing list of packages to add to your project.` Try running `yarn install` instead.
 * Log in using the credentials referred to in the new version of the README. Follow instructions to create new client ID.
